Client/Client.csproj

```
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
+      <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
```

Please change order of versions and add LangVersion:

```suggestion
      <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
      <LangVersion>8</LangVersion>  
```